AprilSylph/XKit-Rewritten
# Quote Replies: Show all buttons when using touchscreen

### Description


This shows the Quote Replies button whether or not the user is hovering over a notification if they're using a device without the ability to hover or if they have made use of the touchscreen since the extension was turned on.

I didn't worry about the fact that Tumblr's block buttons don't have this behavior, but still take up some space, so the spacing does look a bit odd.

I did try this on a Surface device without a keyboard attached, and the media query works in that case, so I think one could make a reasonable argument that the media query alone is sufficient and the on-touch behavior change is unnecessary, and users with touchscreen laptops with trackpads can just use the trackpads. I could go either way on this.

Resolves #895

### Testing steps


- Enable Quote Replies and navigate to the notifications page on a touchscreen-only mobile device. Quote Replies buttons should appear on replies.
- ~~Enable Quote Replies and navigate to the notifications page on a laptop with a touchscreen. Use the touchscreen to interact with the page. Quote Replies buttons should appear on replies.~~ functionality removed
